circuit/src/exit_account.rs

```diff
@@ -34,6 +60,7 @@ impl CircuitFragment for ExitAccount {
     /// Builds a dummy circuit to include the exit account as a public input.
     fn circuit(builder: &mut CircuitBuilder<F, D>) -> Self::Targets {
         let address = builder.add_virtual_hash_public_input();
+        builder.register_public_inputs(&address.elements);
```

Comment:
I don't think this is needed? I would expect that `builder.add_virtual_hash_public_input` automatically exposes the felts as public inputs

Reply:
it doesn't, I checked, it treats it as private if you don't register it

Reply:
That's weird, this is the code it's calling: https://docs.rs/plonky2/latest/src/plonky2/plonk/circuit_builder.rs.html#412-416
So, according to that definition, it definitely _should_ be registering them.

Reply:
my ide routes me here
```
    /// Registers a new `HashOutTarget` as a public input, adding
    /// internally `NUM_HASH_OUT_ELTS` virtual targets.
    pub fn add_virtual_hash_public_input(&mut self) -> HashOutTarget {
        HashOutTarget::from(self.add_virtual_public_input_arr::<4>())
    }
```